src/Sarif.Viewer.VisualStudio/Sarif/SarifLog.extensions.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.CodeAnalysis.Sarif;
+
+namespace Microsoft.Sarif.Viewer.Sarif
+{
+    /// <summary>
+    /// Extension methods for the <see cref="SarifLog"/> class used by the viewer.
+    /// </summary>
+    internal static class SarifLogExtensions
+    {
+        static internal bool HasNoResults(this SarifLog sarifLog)
+        {
+            if (sarifLog.Runs == null)
+            {
+                return false;
+            }
+
+            foreach (Run run in sarifLog.Runs)
+            {
+                if (!run.HasNoResults())
```

Comment:
>!run.HasNoResults() [](start = 20, length = 19)

BEHAVIORAL QUESTION: Do we want to show the bar if _any_ run in the log lacks results, or only if _all_ of them lack results (which is what I'm doing here)?

IMO the behavior here is correct. If any log has a result, then _something_ will show up in the error list, and the user won't wonder if the analysis finished. The idea is to show give the user explicit feedback in case _nothing_ shows up in the error list.

@mikefan